### ISSUE TITLE
Adding newer version of existing mod to the instance should remove the older version or show a warning

### ISSUE THREAD

**Issue by T3sT3ro** [MultiMC/Launcher#5217]
### Role

I play modded MC and use it to manage my instances

### Suggestion

I want multiMC to automatically remove/replace the mod in the instance if I use "Add mod" button to add more recent version of already existing mod. The more recent version of the mod should inherit the replaced mod's enabled/disabled status.

### Benefit

It allows to function better with mod managers that download mods into one directory. It also reduces the amount of manual work I have to do when upgrading an instance.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

I use [ferium](https://github.com/gorilla-devs/ferium/) to manage my mod instances, and when I try to update the instance in multiMC I use the "add mods" button and select all the mods in the directory. 

This way there are often two mods added to the instance - one with the previous revision and one with the more recent. I have to manually remove older mods then.

It would be beneficial if multiMC automatically removed the old mods or at least showed a warning/marked mods that are added to the instance multiple times with different versions.

**Comment by Primitive-Human** [comment]
A suggestion about Ferium and automatically removing older versions of mods has been made.

https://github.com/MultiMC/Launcher/issues/4926

https://github.com/MultiMC/Launcher/issues/4952

However, this won't work with mods made by those who don't understand versioning (Mcreator), or forks of mods that have their own versioning scheme, like Had Enough Items or Mekanism Community Edition. 

At the very least though, a `DUPLICATE` highlight in the Mods screen would be helpful to quickly highlight duplicate mods. I don't want MultiMC to automatically remove those mods though, because it seems MMC permanently deletes files, rather than moving them to a bin folder.

**Comment by T3sT3ro** [comment]
Certainly, the invalid versioning schemes are a problem. Highlight detection could be done be reading MODID metadata by inspecting META-INF and similar loader specific files.
